lib/create-icon-set-from-fontawesome5.js

```diff
@@ -21,7 +21,12 @@ export const FA5Style = {
 
 let shouldSetupFA5 = true;
 
-export function createFA5iconSet(glyphMap, proVersion = false) {
+export function createFA5iconSet(
+  glyphMap,
+  metadata = {},
+  proVersion = false,
+  fallback = true
```

Comment:
Do we need this `fallback` argument if it's always true?

Reply:
This was meant to give more control in case some people want to disable the fallback (for whatever reason). It adds no complexity to normal users but adds some options for those who wish to do this. If it's disabled then the ```brand``` prop is needed (with the fallback it can be omitted). 🙂

Reply:
Honestly don't see the need for the `brand` prop, it's not really a variant, just a different font.

Reply:
Okey, I'll just remove the ```brand``` prop and the fallback option!